meson.build

```diff
@@ -44,6 +44,11 @@ else
     meson.add_dist_script(py, versioneer, '-o', '_version_meson.py')
 endif
 
+cy = meson.get_compiler('cython')
+if cy.version().version_compare('>=3.1.0')
```

Comment:
We may have to bump Cython first for this to get hit. Our build dependency is currently `~=3.0.5`

Reply:
Cython 3.1.0 has not been released yet. However, in #59136 we install Cython nightly to build 3.13 and free-threading wheels which supports this.